app/views/unattended/report_templates/duplicate_hosts.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Duplicate hosts
+snippet: false
+template_inputs:
+- name: hosts
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    for report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+- name: attribute
+  required: true
+  input_type: user
+  description: Host attribute to group by
+  advanced: false
+  value_type: plain
+model: ReportTemplate
+-%>
+<%- report_headers 'Id', input('attribute').capitalize -%>
+
+<%- load_hosts(search: input('hosts'), group_by: input('attribute')).each_record do |host| -%>
+<%- report_row({
+      'Id': host.id,
+      input('attribute').capitalize => host.send(input('attribute'))
```

Comment:
send will not work with safemode, the explicit attributes list may help with this